action.yml

```diff
@@ -104,42 +107,45 @@ runs:
         ret="${ret//$'\r'/'%0A'}"
         echo "::set-output name=prtype::$ret"
 
-    - id: should-proceed
+    - name: Bail when no updates to analyze
+      id: should-proceed
       shell: bash
       if: "contains(steps.get-prtype.outputs.prtype, 'NA')"
       run: |
         echo 'exiting with 0 since package dependency files were not modified'
         echo '0' > $HOME/returncode.txt
 
-    - name: Analyze project lockfile
+    - name: Analyze project lockfile with phylum CLI
       shell: bash
       if: "!contains(steps.get-prtype.outputs.prtype, 'NA')"
       run: |
         export PATH="$HOME/.phylum:$PATH"
         pushd $GITHUB_WORKSPACE || exit 11
-          phylum analyze -l $PHYLUM_LABEL ${{ steps.get-prtype.outputs.prtype }} --verbose --json > ~/phylum_analysis.json
-          echo "[*] Analyzed ${{ steps.get-prtype.outputs.prtype }} under label ${PHYLUM_LABEL} and wrote results to ~/phylum_analysis.json"
+        phylum analyze -l $PHYLUM_LABEL ${{ steps.get-prtype.outputs.prtype }} --verbose --json > ~/phylum_analysis.json
+        echo "[*] Analyzed ${{ steps.get-prtype.outputs.prtype }} under label ${PHYLUM_LABEL} and wrote results to ~/phylum_analysis.json"
         popd
 
-
-    # - name: tmate
-      # uses: mxschmitt/action-tmate@v3
-
-    - name: invoke test matrix
+    - name: Invoke test matrix
       shell: bash
       if: "contains(inputs.invoke_test_matrix, 'true')"
-      run: |
-        python $GITHUB_ACTION_PATH/test_matrix.py
+      run: python $GITHUB_ACTION_PATH/test_matrix.py
 
-    - name: python script analyze.py
+    - name: Compare added dependencies in PR to analysis results
       shell: bash
       if: "!contains(steps.get-prtype.outputs.prtype, 'NA')"
-      run: python $GITHUB_ACTION_PATH/analyze.py "analyze" $GITHUB_REPOSITORY ${{ github.event.number }} ${{ inputs.vul_threshold }} ${{ inputs.mal_threshold }} ${{ inputs.eng_threshold }} ${{ inputs.lic_threshold }} ${{ inputs.aut_threshold }}
-
-    # - name: tmate
-      # uses: mxschmitt/action-tmate@v3
-
-    - id: get-returncode
+      run: >
+        python $GITHUB_ACTION_PATH/analyze.py
+        "analyze"
+        $GITHUB_REPOSITORY
+        ${{ github.event.number }}
+        ${{ inputs.vul_threshold }}
+        ${{ inputs.mal_threshold }}
+        ${{ inputs.eng_threshold }}
+        ${{ inputs.lic_threshold }}
+        ${{ inputs.aut_threshold }}
```

Comment:
Formatting error?

Reply:
No...this is using the YAML folded style (e.g., the `>` character) notation for a multi-line string. All newlines in this folded block will be replaced with a single space. This makes it easier to adhere to line length restrictions and also view command line arguments in their logical groupings...one per line here...which also helps when updating the command to show more specific diffs.

Reply:
Ah, I think I was thinking of `|`. Though I do think that the formatting here could be improved. No indentation between the different lines certainly makes things look a bit odd.

Reply:
Change made.

Reply:
Well...dang. I tested the changes after making the indentation and this command now fails. Looking at some [informal YAML documentation/training](https://learnxinyminutes.com/docs/yaml/) shows that the reason is expected:

```
folded_style: >
    This entire block of text will be the value of 'folded_style', but this
    time, all newlines will be replaced with a single space.

    Blank lines, like above, are converted to a newline character.

        'More-indented' lines keep their newlines, too -
        this text will appear over two lines.
```

Gonna put this back, re-test, and re-request another review...

Reply:
...okay...the re-test was good:
<img width="1400" alt="image" src="https://user-images.githubusercontent.com/18729796/163266071-1db9b7d1-c5dc-4422-962b-aab85b83dd16.png">

Reply:
This is the disadvantage of far-too-complicated markup languages like yaml. So much opportunity for things to behave in an unexpected way.